accname/name/comp_name_from_content.html

```diff
@@ -258,7 +258,39 @@ <h1 data-expectedlabel="Call us" data-testname="heading name from content with t
 <h1 data-expectedlabel="CALL US" data-testname="heading name from content with text-transform:uppercase" class="ex" style="text-transform:uppercase;">Call us</h1>
 <h1 data-expectedlabel="Call Us" data-testname="heading name from content with text-transform:capitalize" class="ex" style="text-transform:capitalize;">Call us</h1>
 <h1 data-expectedlabel="call us" data-testname="heading name from content with text-transform:lowercase" class="ex" style="text-transform:lowercase;">Call us</h1>
-  
+
+<!-- Giacomo Petri first test, be patient if there is any mistake :D -->
+<h1>Name from content containing element with implicit term role</h1>
+<p>This test aims to verify the computed acc name of elements (buttons, headings, links) containing a term role.</p>
+<p>This test covers:</p>
+<ul>
+  <li>Elements with implicit button, heading, link roles containing element with implicit term role</li>
+  <li>Elements with implicit button, heading, link roles containing element with explicit term role</li>
+  <li>Elements with explicit button, heading, link roles containing element with implicit term role</li>
+  <li>Elements with explicit button, heading, link roles containing element with explicit term role</li>
+</ul>
```

Comment:
This level of detail is not necessary, since it's covered already in the comments and test names below.
```suggestion
```